pobieracz_danych_gugik.py

```diff
@@ -1325,37 +1325,46 @@ def model3d_selected_powiat_btn_clicked(self):
         elif self.dockwidget.model3d_lod1_rdbtn.isChecked() and self.dockwidget.model3d_lod2_rdbtn.isChecked():
             standard = ["LOD1", "LOD2"]
         elif not self.dockwidget.model3d_lod1_rdbtn.isChecked() and not self.dockwidget.model3d_lod2_rdbtn.isChecked():
-            msgbox = QMessageBox(QMessageBox.Information, "Ostrzeżenie:",
-                                 f"Nie wybrano standardu")
+            msgbox = QMessageBox(QMessageBox.Information, "Ostrzeżenie:", "Nie wybrano standardu")
             msgbox.exec_()
             return False
 
         od_data_text = self.dockwidget.model3d_dateEdit_comboBox_1.currentText()
         do_data_text = self.dockwidget.model3d_dateEdit_comboBox_2.currentText()
 
-        try:
-            od_data = int(od_data_text)
-        except ValueError:
-            msgbox = QMessageBox(QMessageBox.Warning, "Błąd", "Nieprawidłowa data początkowa.")
-            msgbox.exec_()
-            return False   
-        try:
-            do_data = int(do_data_text)
-        except ValueError:
-            msgbox = QMessageBox(QMessageBox.Warning, "Błąd", "Nieprawidłowa data początkowa.")
-            msgbox.exec_()
-            return False                  
+        def model3d_poprawnosc_dat(od_data_text, do_data_text):
```

Comment:
Zrób z tego proszę funkcję dynamiczną (chodzi o to, żeby tak jak wszystkie inne funkcje mogła być wywoływana z każdego miejsca w kodzie przez self.model3d_poprawnosc_dat) i wyrzuć ją poza funkcję model3d_selected_powiat_btn_clicked